app/assets/javascripts/users.js

```diff
@@ -65,6 +65,11 @@ WebsiteOne.define('Users', function () {
         });
 
         $('#require_users_profile').modal({show: true});
+
+        // Enable bootstrap tooltips on non-touch devices only
+        if (!('ontouchstart' in document.documentElement)) {
+            $('[data-toggle="tooltip"]').tooltip({delay: { "show": 500, "hide": 100 }});
```

Comment:
yes, I also see the tooltip working if I comment out that javascript, so I don't think it is enabling the tooltip:

https://developer.mozilla.org/en/docs/Defining_Cross-Browser_Tooltips

I'm also a little concerned about a comment that describes what a piece of code is supposed to do, and I'm not yet convinced that the code that follows does what is described in the comment.

Either way we should avoid comments and prefer to write method names that accurately describe what code does, e.g. 

```javascript
function addToolTipDelayToNonTouchDevices(){
  if (!('ontouchstart' in document.documentElement)) {
    $('[data-toggle="tooltip"]').tooltip({delay: { "show": 500, "hide": 100 }});
  }
}
```

This reduces the chance of the comment going out of date, and also provides us with methods that can be tested independently